components/movementsensor/last_error.go

```diff
@@ -0,0 +1,25 @@
+package movementsensor
+
+import (
+	"sync"
+)
+
+type lastError struct {
```

Comment:
[nit] getting ahead of this while you're working on it - can we put this in `movementsensor/utils.go`

Reply:
:shrug: if you want. I originally put it there, and then decided that utils.go is more for error reporting than actual utility things, so I moved it out. but I can put it back.